ctwrap/simulation.py

```diff
@@ -1,4 +1,11 @@
-"""Object handling variations."""
+"""This ctwraper module handles batch cantera simulations variations.
+Cantera based modules are pass to this module where they are run in batches.
+see `adiabatic flame example <../examples/adiabatic_flame_example.ipynb>`_ or
+`ignition_example <../examples/ignition_example.ipynb>`_ .
```

Comment:
Make sure those links work once sphinx has processed this. Also, they should point to the notebooks in the pages folder  (the notebooks in examples are redundant and should be removed).